### DISCUSSION TITLE
how to fix this PERM: operation not permitted

### DISCUSSION THREAD

**Discussion by rifkidocs** [illixion/vscode-vibrancy-continued#64] [Q&A] [answered]
hi i got this error
PERM: operation not permitted, unlink 'c:\Users\Administrator\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vscode-vibrancy-runtime-v6\displayconfig.node'

how should i do?

**Comment by illixion** [comment, marked as answer]
Known issue, please check the pinned issue for a workaround: https://github.com/illixion/vscode-vibrancy-continued/issues/58#issue-1590455565

**Reply by illixion** [reply]
Just released an update that includes a partial fix for this issue, so the extension should now work properly without workarounds.
